openshift/openshift-copies.go

```diff
@@ -355,7 +346,9 @@ func validateClusterInfo(clusterName string, clusterInfo clientcmdCluster) []err
 	}
 	if len(clusterInfo.CertificateAuthority) != 0 {
 		clientCertCA, err := os.Open(clusterInfo.CertificateAuthority)
-		defer clientCertCA.Close()
+		defer func() {
+			_ = clientCertCA.Close()
```

Comment:
That's something we could log using `logrus.Debugf("error closing client cert CA: %v", err)`